src/cowrie/ssh_proxy/clientTransport.py

```diff
@@ -0,0 +1,166 @@
+# Copyright (c) 2019 Guilherme Borges <guilhermerosasborges@gmail.com>
```

Comment:
in the filename, we don't usually use capitals in Cowrie, could you change it to 'clienttransport.py', without the capital T?

Reply:
Would an underscore between words be ok?

Reply:
That'd be fine. Or just add the words together, but no capitals please.